.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,27 @@
-Please delete this line and the text below before submitting your contribution.
+# Pull Request checklist
 
----
+Thanks for contributing!
+Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.
 
-Thanks for contributing! If this contribution is for instructor training, please send an email to checkout@carpentries.org with a link to this contribution so we can record your progress. You’ve completed your contribution step for instructor checkout just by submitting this contribution.  
+* * *
 
-Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.  If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact The Carpentries Team at team@carpentries.org.  
+## Tasks
 
----
+- [ ] Read [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md).
+- [ ] Create an [issue](https://github.com/ESMValGroup/tutorial/issues) to discuss what you are going to do.
+- [ ] Give this pull request a descriptive title.
+- [ ] If you are contributing to the lesson materials, make sure the content does not have spelling or grammar errors. See `Lesson development` in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md).
+- [ ] Please use a `markdown lint tool` to check that your files do not contain errors. See `Lesson development` in[CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md).
+- [ ] Preview changes on your own machine before pushing them to GitHub by running `make serve`, alternatively `make docker-serve`. See `Lesson development` in[CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md).
+- [ ] All code instructions have been tested.
```

Comment:
Do we want to make sure that these have been tested by the author and the reviewer?
```suggestion
- [ ] All code instructions have been tested by the PR author.
- [ ] All code instructions have been tested by the reviewer.
```

Reply:
@ledm Thanks. I think that by default both reviewers and authors should check all the items provided in this template. So, we don't need to mention `reviewer` or `author` for this specific item.